bundles/org.openhab.binding.playstation/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:playstation:PS3">
+		<parameter name="ipAddress" type="text" required="true">
+			<context>network-address</context>
+			<label>IP address</label>
+			<description>The local IP address of the PlayStation 3.</description>
+			<advanced>true</advanced>
+		</parameter>
+	</config-description>
+
+	<config-description uri="thing-type:playstation:PS4">
```

Comment:
Can you add these config parameters to the readme?

Reply:
Did you push your changes?

Reply:
I added all configs to the demo thing example, should there be a more expansive description for all config items in the readme?

Reply:
Yes, there should be a table like here https://www.openhab.org/addons/bindings/telegram/#thing-configuration